client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/shipping-rate.js

```diff
@@ -47,15 +47,30 @@ class ShippingRate extends Component {
 				delivery_date_guaranteed,
 				delivery_date,
 			},
-			rateObjectSignatureRequired,
-			rateObject,
 			isSelected,
 			updateValue,
+			signatureRates,
 		} = this.props;
-		let requiredSignatureCost = null;
 		let details = 'Includes tracking';
+
+		const defaultOption = {
+			label: translate( 'No signature required' ),
+			value: false,
+		};
+		const signatureOptions = concat( defaultOption, values(
+			mapValues( signatureRates, ( r, key ) => {
+				const priceString = ( 0 === r.optionNetCost ) ? translate( 'free' ) :
+					translate( '+%s', { args: [ formatCurrency( r.optionNetCost, 'USD') ] } );
```

Comment:
Hardcoding this to USD is acceptable for now, but I prefer adding a TODO here, we might want to support more stuff after multi-carriers

Reply:
I think this is fine for now since there's a bunch of places in the code with 'USD' hardcoded. These should be easy to track down and update if these become configurable later.